src/test/numa.c

```diff
@@ -39,12 +39,21 @@ int main(void) {
   ret = mbind(p, 16 * page_size, MPOL_PREFERRED, NULL, 0, MPOL_MF_MOVE);
   test_assert(ret == 0 || (ret == -1 && errno == ENOSYS));
 
+  // sanity check
   ret = set_mempolicy(0, NULL, 0);
   test_assert(ret == 0);
-
   ret = get_mempolicy(NULL, NULL, 0, NULL, 0);
   test_assert(ret == 0);
 
+  // test in and out params
+  unsigned long nodemask = 0x1;
+  ret = set_mempolicy(MPOL_BIND, &nodemask, 8 * sizeof(nodemask) + 1);
```

Comment:
This is terribly confusing (the parameter should have always been defined to just be sizeof(nodemask)!!!). It seems to me the maximum storable node ID might be 63 for a 64-bit mask, in which case 64 is the right value to pass here. Anyway I think it's safest to pass 8 * sizeof(nodemask) here, for our purposes.